### ISSUE TITLE
[enhancement] Add ability to send a object as a payload

### ISSUE THREAD

**Issue by bricefriha** [bricefriha/CustardApi#22]
Since the client always has to convert an object or array to JSON to pass it to the method, wouldn't sending the array or object directly make more sense?

#### Current 
``` Csharp
 // Arrange
 var userToCreate = new ReqresUser
 {
     Name = "Jackson",
     Job = "Writer content",
 };
 string action = "users";
 string controller = "api";

 // Act
 var result = await _serviceReqres.Post<ReqresUser>(controller, action: action, jsonBody: JsonConvert.SerializeObject(userToCreate));
```

#### Proposal
``` Csharp
 // Arrange
 var userToCreate = new ReqresUser
 {
     Name = "Jackson",
     Job = "Writer content",
 };
 string action = "users";
 string controller = "api";

 // Act
 var result = await _serviceReqres.Post<ReqresUser>(controller, action: action, payload: userToCreate);
```

**Comment by bricefriha** [comment]
Closed by #28 .

**Comment by bricefriha** [comment]
this feature will be available in the upcoming v0.4

**Comment by bricefriha** [comment]
PR redone #31
